src/pages/console/uibuilder/figmatocode.mdx

```diff
@@ -87,13 +87,26 @@ Although you can inspect the component code, any changes to component code will
 
 ## Upgrading Figma files
 
+In order to upgrade your Figma file, the newest version of all Amplify UI primitives must be imported into your customized Figma file.
+
 1. Duplicate the new [Figma community file](https://www.figma.com/community/file/1047600760128127424) into your Figma account.
 ![Duplicate the new figma file](/images/console/ui-figma-file.png)
-1. (optional) Delete all the components in the 'My Components' page of the new Figma file.
-1. Copy everything in the 'My Components' page of your old Figma file and paste it into the 'My Components' page of the new Figma file.
-1. Go through each component on the 'My Components' page and update the broken component instances to point to the local components on the 'Primitives' page. Unfortunately, there is no easy way to bulk update component instances in Figma. **This is important because if you do not do this, your design might look ok, but the React code it generates won't use the [Amplify UI components](https://ui.docs.amplify.aws).**
+1. Copy everything in the 'README' page of the newest community file into your customized Figma file
+    * If your Figma file doesn't have a 'README' page, create new page named README
+1. Delete all content in the 'Primitives' page of your customized Figma file
+1. Copy everything in the 'Primitives' page of the new Figma file, and paste it into the 'Primitives' page of your customized Figma file.
```

Comment:
Feel this can be worded to prevent any confusion for users. For example: change `new figma file` to like `cloned figma file`

Reply:
Added this language